onnx_tf/frontend.py

```diff
@@ -183,6 +185,21 @@ def tensorflow_graph_to_onnx_graph(cls, graph_def, output, name="graph"):
                       output_proto,
                       consts_proto)
 
+  @classmethod
+  def tensorflow_graph_to_onnx_model(cls,
+                                     graph_def,
```

Comment:
Never be called, is it ok?

Reply:
I imagine that this function is left for users to call.
And it looks like everyone else is doing a command-line interface for model conversion so I'm considering making a command line wrapper to call this function.

But let me export this function nevertheless.